scripts/train.py

```diff
@@ -45,6 +45,11 @@ def create_models(num_classes, weights='imagenet', multi_gpu=0):
     # Keras recommends initialising a multi-gpu model on the CPU to ease weight sharing, and to prevent OOM errors.
     # optionally wrap in a parallel model
     if args.multi_gpu > 1:
+
+        if args.batch_size < args.multi_gpu:
+            raise ValueError("Batch-Size(%i) must be equal or higher than number of GPUs(%i)" %
+                             (args.batch_size, args.multi_gpu))
```

Comment:
Could you use `str.format` instead of the deprecated `%` formatting? Other than that, looks good!

Reply:
Whoops! Old habits ;)